java/arcs/android/devtools/DevToolsMessage.kt

```diff
@@ -0,0 +1,20 @@
+package arcs.android.devtools
+
+/**
+ * An interface for messages between [DevToolsService] and the client.
+ */
+interface DevToolsMessage {
+
+    /** The type of DevToolsMessage */
+    val kind: String
+    /** The message to be passed */
+    val message: String
+
+    /** Return a JSON string that can be passed to the client. */
+    fun toJson(): String
+
+    /** Track message types */
+    companion object {
+        val RAW_MESSAGE: String = "RawStoreMessage"
+    }
+}
```

Comment:
Formatting nit:

```suggestion
interface DevToolsMessage {
    /** The type of DevToolsMessage */
    val kind: String
    
    /** The message to be passed */
    val message: String

    /** Return a JSON string that can be passed to the client. */
    fun toJson(): String

    /** Track message types */
    companion object {
        val RAW_MESSAGE: String = "RawStoreMessage"
    }
}
```

 Also, what is `RAW_MESSAGE` intended to be used for? (a comment would be helpful)

Reply:
Updated and added a comment.